docs/references/mqtt-namespace.md

```diff
@@ -828,6 +829,13 @@ The container JSON message is comprised of the following elements:
 
 * Type: denotes the type of inventory payload
 
+* State: describes the container's current state
+     * Active: Container is running
+     * Installed: Container is starting
+     * Uninstalled: Container has failed, or is stopped
+     * Unknown: Container state can not be determined
+
+
```

Comment:
I would format like this:

```suggestion
     * `active`: Container is running
     * `installed`: Container is starting
     * `uninstalled`: Container has failed, or is stopped
     * `unknown`: Container state can not be determined

```

because this is how you receive them when performing a request. What do you think?